backend/api/communication/src/main/java/co/airy/core/api/communication/Stores.java

```diff
@@ -217,7 +217,9 @@ public void storeReadReceipt(ReadReceipt readReceipt) throws ExecutionException,
     }
 
     public void storeMetadata(Metadata metadata) throws ExecutionException, InterruptedException {
```

Comment:
Nitpick: This method is also used in other places. I'm not sure if we want to introduce metadata value "" == delete as a **global** convention. Could you here maybe add a `deleteMetadata`? :)

Reply:
Not sure if saving empty strings `""` on the store is useful anyway. I am missing something?

Reply:
True. Perhaps we can keep this until someone complains about their empty strings not being saved 👍